backend/app/Repositories/Contracts/TableVisitorsRepository.php

```diff
@@ -16,5 +16,5 @@ public function groupByBrowser(int $website_id, string $from, string $to): Colle
 
     public function groupByOperatingSystem(int $website_id, string $from, string $to): Collection;
 
-    public function groupByScreenResolution(int $website_id, string $from, string $to): Collection;
+    public function groupByScreenResolution(int $website_id, string $from, string $to, bool $newVisitors = false): Collection;
```

Comment:
never add flags as parameters

Reply:
it is the worst practice developers can do. If you add flag as parameter it means that your method will be responsible about two things at least. It is a very rude violation of single responsibility

Reply:
Fixed